sonic_y_cable/y_cable.py

```diff
@@ -75,6 +80,15 @@
 SWITCH_COUNT_MANUAL = "manual"
 SWITCH_COUNT_AUTO = "auto"
 
+FIRMWARE_INFO_PAYLOAD_SIZE = 48
+NUM_MCU_SIDE = 3
+
+EEPROM_READ_DATA_INVALID = -1
+EEPROM_ERROR = -1
+EEPROM_TIMEOUT_ERROR = -1
+
+WAIT_TIMEOUT_FOR_BER_EYE = 1
```

Comment:
Suggest breaking this into two, one for BER, one for eye, and also add the units in the name (e.g., `BER_TIMEOUT_SECS`, `EYE_TIMEOUT_SECS`).

Reply:
fixed